eureka-common/src/main/java/edu/emory/cci/aiw/cvrg/eureka/common/props/AppProperties.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package edu.emory.cci.aiw.cvrg.eureka.common.props;
+
+
+import com.google.inject.Singleton;
+/**
+ *
+ * @author Miao Ai
+ */
+public class AppProperties extends AbstractProperties {  
```

Comment:
This functionality is already in ServiceProperties, and the Resource class you created is in eureka-services, so just use ServiceProperties.